UpcomingMovies/UpcomingMovies/UpcomingMovies/Settings/Settings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UpcomingMovies.Settings
+{
+    public static class Settings
+    {
+        public static string ApiKey { get; } = "1f54bd990f1cdfb230adb312546d765d";
```

Comment:
Just a syntax sugar :), you can write  `ApiKey => "1f54bd990f1cdfb230adb312546d765d"